### PR TITLE
[cookbook]: Chat With PDF example

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR was unintentionally merged from #21 and then reverted. So this is a fresh Push with all the suggested changes.

### CODE REVIEW

**Comment by PremAIDevRel** [comment]
@Anindyadeep Looking good! This just needs some spelling and grammar corrections before we merge. We can go over it in our meeting later.

File: cookbook/chat-with-pdf.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
+
+
+Welcome to Prem AI cookbook section. In this first example, we will ste by step build a simple chat with pdf application (~ 100 lines of code) using [prem ai sdk](/get-started/sdk.mdx) and [streamlit](https://streamlit.io/). The best part, you don't even need to understand complex mechanisms like RAG or Retrieval Augmented Generations to make this app. So without further adue, let's get started. You can find the full code [here](https://github.com/premAI-io/cookbook/chat-with-pdf).
```
Comment (PremAIDevRel) [inline cookbook/chat-with-pdf.mdx:3]: Correction: Welcome to the Prem AI cookbook section. In this first example, we will step by step build a simple chat with a PDF application (~ 100 lines of code) using the Prem AI SDK and Streamlit. The best part is that you don't even need to understand complex mechanisms like RAG or Retrieval Augmented Generations to make this app. So, without further ado, let's get started. You can find the full code here.

Comment (Anindyadeep) [reply cookbook/chat-with-pdf.mdx]: Done

File: images/cookbook/chat_with_pdf_app.png
Comment (nsosio) [inline images/cookbook/chat_with_pdf_app.png:1]: the limit per file is 10MB and not 200MB as per image

File: cookbook/chat-with-pdf.mdx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+```python
+import time 
+import json 
+import premai 
+import streamlit as st 
+
+premai_api_key = st.secrets.premai_api_key
+premai_project_id = 4071
+premai_repository_id = 2100
```
Comment (nsosio) [inline cookbook/chat-with-pdf.mdx:47]: avoid real IDs

Comment (Anindyadeep) [reply cookbook/chat-with-pdf.mdx]: Done

File: images/cookbook/chat_with_pdf_app.png
Comment (Anindyadeep) [reply images/cookbook/chat_with_pdf_app.png:1]: Done

**Review by nsosio — APPROVED** [review]

**Comment by nsosio** [comment]
@Anindyadeep there are some conflicts

**Comment by Anindyadeep** [comment]
> @Anindyadeep there are some conflicts

Done

**Pull request merged**